baseplate/sidecars/live_data_watcher.py

```diff
@@ -128,25 +90,76 @@ def on_change(self, data: Optional[bytes], _znode_stat: ZnodeStat) -> None:
             if self.owner and self.group:
                 os.fchown(tmpfile.fileno(), self.owner, self.group)
             os.fchmod(tmpfile.fileno(), self.mode)
-            try:
-                json_data = json.loads(data.decode("UTF-8"))
-            except json.decoder.JSONDecodeError:
-                # If JSON fails to decode, still write the bytes data since
-                # we don't necessarily know if the the contents of the znode
-                # had to be valid JSON anyways.
-                tmpfile.write(data)
-            else:
-                # If no exceptions, we have valid JSON, and can parse accordingly.
-                data = NodeWatcher.get_data_to_write(json_data, data)
-                if data is None:
-                    logger.warning(
-                        "No data written to destination file. Something is likely misconfigured."
-                    )
-                    return
-                tmpfile.write(data)
+
+            tmpfile.write(data)
         os.rename(self.dest + ".tmp", self.dest)
 
 
+def _parse_loader_type(data: bytes) -> LoaderType:
+    try:
+        json_data = json.loads(data.decode("UTF-8"))
+    except (UnicodeDecodeError, json.decoder.JSONDecodeError):
```

Comment:
I ran the following against a copy of our zk instance and found `UnicodeDecodeError`, otherwise we're clean to load everything in there (even the things we don't actually intend to load)
 
```py
zk = kazooClient('ourcluster')
def parse_recursive(start='/'):
     print(f'Checking {start}')
     children  = zk.get_children(start)
     for c in children:
         parse_recursive(start + '/' + c)
     data, _ = zk.get(start)
     try:
         _parse_loader_type(data)
     except Exception as e:
         print(f'FAIL FAIL FAIL FAIL FAIL {e.__class__.__file__}: {str(e)} \n {data} \n')
         
   parse_recursive()
   ```